### PR TITLE
World builder pc-> development

### PR DESCRIPTION
**Pull request by FlorisVideler**

### CODE REVIEW

File: tardis-world/index.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html>
     <head>
+        <script src="../scripts/browserMqtt.js" type="text/javascript"></script>
```
Comment (DaanWesterhof) [inline tardis-world/index.html]: These are already included, wont this cause issues?

**Review by DaanWesterhof — CHANGES_REQUESTED** [review]
I dont think the commit at the tardis world index is needed. Code in main.js looks great, but the file is VERY large and i think it can be split into atleast 2 or 3 files, so the code will be easier to digest for other developers and its clearer what code does what.

**Comment by FlorisVideler** [comment]
Will do

**Review by DaanWesterhof — CHANGES_REQUESTED** [review]
looks good, still need to remove the addition in the tardis-world/index.html of src="../scripts/browserMqtt.js" and src="../scripts/mqtt_client.js"
otherwise perfect

**Pull request merged**